frontend/ui/subview.jsx

```diff
@@ -43,15 +43,31 @@ const mapStateToProps = (state, ownProps) => {
   return { measures: [] };
 };
 
+const getOptionalTimeWindow = props => {
+  const urlParams = new URLSearchParams(props.location.search);
+
+  return {
+    timeWindow: urlParams.get('window') ? urlParams.get('window') : 'adjusted',
```

Comment:
You could simplify this to just

```urlParams.get('window') || 'adjusted'`

I'll make this adjustment and then land, as otherwise this looks great!